### PR TITLE
Feature/seq2seq data embs

### PR DESCRIPTION
**Pull request by elboy3**
- Update OD and SemSeg to use `support_data_emb`
- Use `support_emb` where previously hard coding task type
- Small S2S bug fix

### CODE REVIEW

File: dataquality/loggers/data_logger/seq2seq.py [hunk 1]
```diff
@@ -219,6 +220,17 @@ def separate_dataframe(
 
         emb = df_copy[emb_cols]
         data_df = df_copy[other_cols]
-        data_df.rename("text", "input")
-        data_df.rename("label", "target_output")
+        data_cols = data_df.get_column_names()
+        if "text" in data_cols:
+            data_df.rename("text", "input")
+        if "label" in data_cols:
+            data_df.rename("label", "target_output")
```
Comment (elboy3) [inline dataquality/loggers/data_logger/seq2seq.py:227]: this was a small bug fix for multiple epochs

**Review by bogdan-galileo — COMMENTED** [review]
I don't think data embs are set in IC either. Do we have to set the property to False too ?

**Comment by dcaustin33** [comment]
> I don't think data embs are set in IC either. Do we have to set the property to False too ?
Yeah we should set it to false.
Tests @elboy3 ?

**Review by dcaustin33 — APPROVED** [review]

File: dataquality/loggers/data_logger/seq2seq.py [hunk 2]
```diff
@@ -219,6 +220,17 @@ def separate_dataframe(
...
         return BaseLoggerDataFrames(prob=prob, emb=emb, data=data_df)
+
+    @property
+    def support_embs(self) -> bool:
```
[the remark this answers is not in the document]
Comment (elboy3) [reply dataquality/loggers/data_logger/seq2seq.py:231]: we support data embs in s2s

**Comment by elboy3** [comment]
> I don't think data embs are set in IC either. Do we have to set the property to False too ?

data embs are already false in IC

**Pull request merged**